### ISSUE TITLE
Bug: does not give warning about lack of aria-label or aria-labelledby on role="img" element(ARIA 1.2 )

### ISSUE THREAD

**Issue by Zidious** [dequelabs/axe-core#3153]
Ref: https://github.com/dequelabs/axe-core-npm/issues/316

[ARIA 1.2: `img`](https://www.w3.org/TR/wai-aria-1.2/#img) - In order for elements with a role of `img` to be perceivable, authors MUST provide a label using the `aria-label` or `aria-labelledby` attribute.
